assets/locales/service.en.toml

```diff
@@ -164,6 +164,11 @@ one = "Skip to page content"
 description = "Variables"
 one = "Variables"
 
+[VariablesExplanation]
+description = "What do the variables mean?"
+one = "What do the variables mean?"
```

Comment:
Minor: Why isn't the translation for one variable singular and why aren't we using the singular if there is only one variable? so... `one = "What does the variable mean?"`

Reply:
It's a good question. The copy has not been confirmed yet and I am using it to remove the question mark for screen readers. I'll push the singular and plural changes